### PR TITLE
Fixed issue #324

### PR DESCRIPTION
**Pull request by arjunpandey2003**

### CODE REVIEW

**Comment by saarthakchats** [comment]
It looks good but please squash the commits!

**Review by 0x48piraj — APPROVED** [review]
That's a broken PR.

**Review by saarthakchats — APPROVED** [review]
LGTM

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Sorry. It's not looking good at all. 
Don't change the `index.html`
Revert your changes please.
And, use `video.yml` to add your videos.

**Review by saarthakchats — CHANGES_REQUESTED** [review]
This is not where you were supposed to make the changes.

**Comment by 0x48piraj** [comment]
Don't change `index.html` @arjunpandey2003
image

**Review by saarthakchats — CHANGES_REQUESTED** [review]
Still not solved

File: index.html [hunk 1]
```diff
@@ -408,12 +408,12 @@ <h4 class="uppercase mb16">Videos</h4>
 				<div class="row vid-row">
 			{% endif %}
 				<div class="col-md-3">
-					<a href="{{video.url}}" target="_blank">
+					<a href="https://www.youtube.com/watch?v=7eg9YnYuK7c&feature=youtu.be" target="_blank">
```
Comment (vaibhavsingh97) [inline index.html:411]: Please don't change `index.html`

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Please revert changes in `index.html`

File: _data/video.yml [hunk 2]
```diff
@@ -3,3 +3,8 @@
   title: About FOSSASIA
   img: FOSSASIA.jpeg
   author_name: Supun Tharinda Edirisuriya
+  
+- url: https://www.youtube.com/watch?v=7eg9YnYuK7c&feature=youtu.be
```
Comment (anshumanv) [inline _data/video.yml:7]: Video is unavailable.

**Comment by ParthS007** [comment]
closing this as a duplicate of #405

**Pull request closed without merging**